src/main.cc

```diff
@@ -359,7 +357,7 @@ int main (int argc, char *argv[])
   
   /* Startup output */
   if ( !( flags & XBOARD ) && ( !opt_quiet ) && ( !opt_uci) ) {
```

Comment:
Taking true branch.  - flags &amp; 1024 evaluates to false.  - opt_quiet evaluates to false.  - opt_uci evaluates to false.